test/utils/image/manifest.go

```diff
@@ -108,6 +108,8 @@ const (
 	AuditProxy
 	// BusyBox image
 	BusyBox
+	// BusyBox image with default user 1234
```

Comment:
```suggestion
	// BusyBoxUser is the BusyBox image with default user 1234
```

I think the above change will fix the golint exception: `comment on exported const BusyBoxUser should be of the form "BusyBoxUser ..."`.